key-chord.el

```diff
@@ -337,15 +353,17 @@ Please ignore that."
               (when (bound-and-true-p eldoc-mode)
                 (eldoc-pre-command-refresh-echo-area))
 
-	      (sit-for delay 0 'no-redisplay))
+	      (sit-for delay 'no-redisplay))
 	    (progn
-	      (setq key-chord-last-unmatched nil)
+	      (setq key-chord-last-unmatched nil
+		    key-chord-idle-state     nil)
```

Comment:
I am an "aligner" myself, but IMO you are overdoing it here and below.

Reply:
OK, fix it.